### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by emmatrain**
Main structure with nav bar and about section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		   <div class="content">
+			   
+<!-- NAVIGATION -->
+			   
+			   <nav>
+					 <li><a href="index.html">01. Home</a></li>
+					 <li><a href="#about">02. About</a></li>
+					 <li><a href="#projects">03. Projects</a></li>
+					 <li><a href="#contact">04. Get in touch</a></li>
+			 </nav>
```
Comment (xz-dice) [inline index.html:31]: Standardise indenting.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<!-- ABOUT SECTION -->
+
+	  <section id="about">
+		  	
+			<a href="#top"><img src="up-arrow.png" class="home-arrow" alt="Arrow pointing up"></a>
+	
+			<div class="bio-content">
+				<h2>About.</h2>
+				<p>	
+					I am a student at IO Academy, a global top 10 school for full stack development.<br><br>After graduating in June 2021, I will be a qualified full stack software engineer and Scrum Master, proficient in 					HTML, CSS, Javascript, PHP and Node.js, with experience in databases and frameworks, Git, unit testing and pair programming.<br><br>I’m ambitious and proactive, with a results-driven and customer-first 					mindset from my experience in sales and marketing. A qualified digital marketer, with skills in SEO, email marketing and social media analytics and an excellent communicator - I am both a confident public 					speaker and articulate writer.<br><br>When I’m not coding, you’ll find me on my yoga mat, wandering Bermuda’s beaches or with my nose in a wine book.<br><br>
```
Comment (xz-dice) [inline index.html:54]: Wrap this paragraph to make it more readable.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+header h1 span:hover {
+	font-family: 'Londrina Solid', serif;
+}
+
+header h2, h3 {
+	font-family: 'Montserrat', sans-serif;
+	color: #FFFFFF;
+}
+
+header h2 {
```
Comment (xz-dice) [inline styles.css:85]: I think you can make this more semantic by removing 'header'?

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+	font-size: 2rem;
+}
+
+header h3 {
```
Comment (xz-dice) [inline styles.css:89]: I think you can make this more semantic by removing 'header'?

**Review by xz-dice — COMMENTED** [review]
All looks good! Happy to look over again for your next review :)

**Review by emmatrain — COMMENTED** [review]
resolved

**Pull request merged**